<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generadores y Pruebas de Numeros Pseudoaleatorios</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Generador de Numeros Pseudoaletorios</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Semilla:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSemilla">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>221</width>
       <height>181</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>65</number>
     </attribute>
     <column>
      <property name="text">
       <string>Conjunto r(i)</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="txtcantidad">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cantidad de numeros:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_importar_datos">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Importar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Exportar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elegir Algoritmo:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbogeneradores">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Cuadrados medios</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Productos medios</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multiplicador constante</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lineal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Congruencial Multiplicativo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Congruencial Aditivo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Congruencial cuadratico</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blum Blum y Shub</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnGenerar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Generar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSemilla2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_vergrafica">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Graficar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_X">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_Y">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>320</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cantidad de decimales:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtcantidad_decimales">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>661</width>
      <height>671</height>
     </rect>
    </property>
    <property name="title">
     <string>Pruebas</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Prueba de Medias:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Prueba de Varianza:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Pruebas de Uniformidad:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Pruebas de Independencia:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>280</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Corridas observadas co:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblco">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>280</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbluco">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>300</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Valor esperado uco:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbloco">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>320</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>320</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>numero de corridas oco:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>340</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Z calculado:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblz">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>340</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>270</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>En conclusion:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAplicarPrueba">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Aplicar arriba y abajo</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtresultado">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>290</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAplicarMedias">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Aplicar</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPrueba_arrabaj_media">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Aplicar con la media</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblmedia">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Media : </string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_li">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>LI : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>LS : </string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_ls">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>451</width>
       <height>151</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>65</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="btn_chicuadrada">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Aplicar Chi Cuadrada</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtresultado_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>En conclusion:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Prueba de Poker:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>0</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Varianza:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblmedia_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>LI : </string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_ls_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_PruebaVarianza">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Aplicar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>70</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>LS : </string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_li_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>331</width>
       <height>151</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>65</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="btnPrueba_Poker">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Aplicar</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtresultado_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>610</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblchicalc">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>600</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>610</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>chi calculado : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>630</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>chi tabla : </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblchitabla">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>630</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblmensaje">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>440</y>
       <width>311</width>
       <height>151</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>65</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="btnPrueba_Huecos">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>370</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Aplicar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>380</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Prueba de Huecos:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblchitabla_3">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>630</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>610</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>chi calculado : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>630</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>chi tabla : </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblchicalc_3">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>600</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtresultado_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>400</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>chi calculado : </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblchitabla_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>chi tabla : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrueba_Series">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Prueba de Series:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>65</number>
    </attribute>
   </widget>
   <widget class="QTextBrowser" name="txtresultado_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblchicalc_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInicio">
    <property name="title">
     <string>Inicio</string>
    </property>
   </widget>
   <addaction name="menuInicio"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
